CommonData/columnutils.h

```diff
@@ -3,30 +3,44 @@
 
```

Comment:
```suggestion

#include <functional>
```
I cannot compiling it without this with MSVC, and buildBot on Windows also failed.

Reply:
Ill have a look